sdk/identity/Azure.Identity.Broker/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Preview support for Proof of Possession (PoP) tokens for `InteractiveBrowserCredential` has migrated out of Azure.Core.Experimental to Azure.Core. This feature is enabled via the `IsProofOfPossessionEnabled` property on `TokenRequestContext`.
```

Comment:
This makes it sound like the feature is still in preview.  Is the feature GA now?  (or .. will be GA when the Azure.Identity library releases?)

Reply:
Yes, it will GA. I'll remove the word `Preview`.